### PR TITLE
[change] Changed wording of "unsubscribe" button #136

### LINKED ISSUE #136: [change] Change wording of "unsubscribe" button
I think we could rename this to "Silence notifications", what do you think @pandafy?

### PR DESCRIPTION
**Pull request by Aryamanz29**
- Changed wording of `unsubscribe-btn`.
- Updated notification image in docs.

Screenshot from 2022-04-13 17-13-20 Screenshot from 2022-04-21 13-42-53

Closes #136

### CODE REVIEW

**Review by nemesisdesign — COMMENTED** [review]
Looks good, thanks @Aryamanz29! Will test shortly.

File: README.rst [hunk 1]
```diff
@@ -626,7 +626,7 @@ may be re-created automatically if the system needs it.
 Silencing notifications for specific objects temporarily or permanently
 -----------------------------------------------------------------------
 
-.. image:: https://github.com/openwisp/openwisp-notifications/raw/docs/docs/images/silence-notifications.png
+.. image:: https://user-images.githubusercontent.com/56113566/163180707-78abfb33-22d3-4265-950e-4c47fa348891.png
```
Comment (nemesisdesign) [inline README.rst:629]: what's going on here?

File: openwisp_notifications/static/openwisp-notifications/js/object-notifications.js [hunk 2]
```diff
@@ -183,6 +183,6 @@ function updateObjectNotificationHelpText($, validTill) {
 
     $('#ow-object-notify > span.ow-icon').removeClass('ow-object-notify-bell');
     $('#ow-object-notify > span.ow-icon').addClass('ow-object-notify-slash-bell');
-    $('#ow-unsubscribe-label').html('Unsubscribed');
+    $('#ow-silence-label').html('Silenced notifications');
```
Comment (nemesisdesign) [inline openwisp_notifications/static/openwisp-notifications/js/object-notifications.js:186]: I would use "Unslience notifications" here

**Review by nemesisdesign — CHANGES_REQUESTED** [review]

File: README.rst (hunk 1 above)
Comment (Aryamanz29) [reply README.rst:629]: I'm trying to change [this images](https://github.com/openwisp/openwisp-notifications/raw/docs/docs/images/silence-notifications.png) with [new one](https://user-images.githubusercontent.com/56113566/163180707-78abfb33-22d3-4265-950e-4c47fa348891.png) using github-user-content.
Just found that we have a separate [docs branch](https://github.com/openwisp/openwisp-notifications/tree/docs) for managing docs image. So it's better to changes there.

Comment (codesankalp) [reply README.rst]: @Aryamanz29 I have uploaded your image in docs branch: https://github.com/openwisp/openwisp-notifications/commit/c829a4316b952a0460e9436c74e3404c6738581c

You can change the url now. :+1:

**Review by nemesisdesign — APPROVED** [review]

**Pull request merged**